src/index.js

```diff
@@ -1,11 +1,21 @@
-const express = require('express');
-const morgan = require('morgan');
-const cors = require('cors');
+import express from 'express';
+import morgan from 'morgan';
+import cors from 'cors';
+import { json, urlencoded } from 'body-parser';
+
+import authRoutes from '../routes/auth';
+import { getDb } from './database';
 
 const app = express();
 
 app.use(morgan('dev'));
 app.use(cors());
+app.use(json());
+app.use(urlencoded({
+  extended: true,
+}));
+
+app.use('/auth', authRoutes);
 
 const port = process.env.PORT || 8000;
```

Comment:
@Saumya29 use this in constants file and import from there.

Reply:
What should I put in constants file? I didn't understand.